### PR TITLE
proj: digital wall calendar and photoframe

### PR DESCRIPTION
**Pull request by ric96**
A digital wall calendar and photoframe written in shell and python
using touchscreen for inteface

Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+# Table of Contents
+
+- [1) Hardware](#1-hardware)
+   - [1.1) Hardware Requirements](#11-hardware-requirements)
+   - [1.2) Hardware Setup](#12-hardware-setup)
+- [2) Software](#2-software)
+   - [2.1) Install Dependencies](#21-install-dependencies)
+   - [2.2) Code Customization Options](#22-code-customization-options)
+   - [2.3) Running the Digital Calendar and Photo frame](#23-running-the-digital-calendar-and -photo-frame)
```
Comment (Mani-Sadhasivam) [inline README.md]: toc link is not shown properly!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+# 1) Hardware
+
+## 1.1) Hardware Requirements
+
+- [Any 96Boards CE](https://www.96boards.org/products/ce/)
+- [HDMI + USB Touchscreen](https://www.arrow.com/en/products/96boards-display-7/linksprite-technologies-inc)
+
+## 1.2) Hardware Setup
+
+- Make sure the touchscreen is plugged in.
```
Comment (Mani-Sadhasivam) [inline README.md]: Better say, "Make sure the touchscreen is connected to 96Boards CE using HDMI cable and USB"?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+2. **Add Images**
+  - On another computer/smartphone connected to the same WiFi network, open a web browser and navigate to ```<ip address of the board>:9090``` and start uploading images.
+3. **Finally Run:**
+```
+$ python wall.py
+```
+or, to run in the background
+```
+$ python wall.py &
+```
```
Comment (Mani-Sadhasivam) [inline README.md:82]: It would be better to add a section about expected output and add some screenshots.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+- [Any 96Boards CE](https://www.96boards.org/products/ce/)
+- [HDMI + USB Touchscreen](https://www.arrow.com/en/products/96boards-display-7/linksprite-technologies-inc)
+
+## 1.2) Hardware Setup
+
+- Make sure the touchscreen is plugged in.
+
+# 2) Software
+
+**This guide assumes that [Debian OS is running on a Dragonboard410c](https://www.96boards.org/documentation/consumer/dragonboard410c/downloads/debian.md.html) on all 4 nodes. How ever the instructions hold true for other 96Boards CE Boards running Debian.**
```
Comment (Mani-Sadhasivam) [inline README.md]: 4 nodes? So we need 4 DB410c's?

**Review by Mani-Sadhasivam — CHANGES_REQUESTED** [review]
Couple of minor suggestions!

File: README.md (hunk 1 above)
Comment (ric96) [reply README.md]: ack

File: README.md (hunk 2 above)
Comment (ric96) [reply README.md]: ack

File: README.md (hunk 3 above)
Comment (ric96) [reply README.md:82]: I'll add in words, not much to see. I'll add a video at the end once that's published. But I need to repo to be published first to add to the video.

**Review by Mani-Sadhasivam — APPROVED** [review]

**Pull request merged**